x-pack/legacy/plugins/lens/public/editor_frame_plugin/merge_tables.ts

```diff
@@ -40,26 +41,27 @@ export const mergeTables: ExpressionFunction<
   context: {
     types: ['kibana_context', 'null'],
   },
-  fn(ctx, { layerIds, tables }: MergeTables) {
+  fn(ctx, { layerIds, tables, _forceNowForTesting }: MergeTables) {
```

Comment:
I don't think we should put test-specific code into our production code (`_forceNowForTesting`). Instead, we ought to mock out `dateMath.parse` and have the mock call the underlying library with `{ forceNow: now }`.

Reply:
I agree. This will also make the test less brittle.